src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -224,17 +225,28 @@ class JesBackend extends Backend with LazyLogging {
   def generateJesOutputs(backendCall: BackendCall): Seq[JesOutput] = {
     val wdlFileOutputs = backendCall.call.task.outputs flatMap { taskOutput =>
       taskOutput.expression.evaluateFiles(backendCall.lookupFunction, new NoFunctions, taskOutput.wdlType) match {
-        case Success(wdlFiles) => wdlFiles.map(_.valueString)
+        case Success(wdlFiles) =>
+          val x = wdlFiles map mapWdlValueGcsPath
```

Comment:
While the functional brethren in the scala community love x's and y's these might not past the @mcovarr test. Although why bother w/ the intermediates in the first place? The double map would be fine here.